lib/kubernetes-deploy/runner_task.rb

```diff
@@ -68,11 +68,8 @@ def validate_configuration(task_template, args)
       begin
         @kubeclient.get_namespace(@namespace) if @namespace.present?
       rescue KubeException => e
-        errors << if e.error_code == 404
-          "Namespace was not found"
-        else
-          "Could not connect to kubernetes cluster"
-        end
+        msg = e.error_code == 404 ? "Namespace was not found" : "Could not connect to kubernetes cluster"
```

Comment:
Rubocop was asking for crazy non-standard indentation on the old version, seemingly because of a problem recognizing the left margin when the return value of the `if` is being concatenated. As much as I dislike changing valid code to please robots, I wasn't able to quickly find the relevant rubocop setting and didn't want to waste time on it. Ultimately, this conditional will be replaced with rescuing two different exceptions once we have a version of kubeclient with [kirs's patch](https://github.com/abonas/kubeclient/pull/233) included.